src/types.ts

```diff
@@ -54,7 +54,7 @@ export type ArNSInteraction = {
   input: PstInput | undefined;
   height: number;
   owner: string;
-  sortKey: string;
+  sortKey?: string;
```

Comment:
Why is this optional?

Reply:
warp has it as optional in their type - but since we compute it ourselves we can set it as non-optional. updated in [98d2433](https://github.com/ar-io/arns-service/pull/76/commits/98d24339b70bf0a4a4850f269d59164c55794922)